tests/service/ExtensionTypingProviderTests.fs

```diff
@@ -15,13 +18,65 @@ open FSharp.Compiler.AbstractIL.Internal.Library
 open FSharp.Compiler.Service.Tests.Common
 open Microsoft.FSharp.Core.CompilerServices
 
+module internal XmlProxyProvidersProject = 
+    let fileName1 = Path.ChangeExtension(Path.GetTempFileName(), ".fs")
```

Comment:
Could it use something like the following?
https://github.com/dotnet/fsharp/blob/ad112ad140717481d86e460fe8a646b5c4c17c47/tests/service/Common.fs#L170